### ISSUE TITLE
[Pico Neo2] The UI wrongly slides from a low view to the center when opening/resuming the app

### ISSUE THREAD

**Issue by Softvision-RemusDranca** [MozillaReality/FirefoxReality#3210]
**[Affected Versions]:**
- Firefox Reality version 10-141130836 (d3f3c9b).

**[Affected Platforms]:**
- Pico Neo2 - Android 8.1.0

**[Prerequisites]:**
- The Firefox Reality app is installed.

**[Steps to reproduce]:**
1. Open the app an observe the UI.

**[Expected results]:**
- The UI is correctly rendered.

**[Actual results]:**
- The UI slides from a low view to the center.

**[Notes]:**
- This issue is also reproducible when the app is quited and resumed.
- Attached is a screen recording of the issue: [link](https://drive.google.com/open?id=1SMheiQYRrZsdxyH4BEUTZL8cT4cFqm_C).

**Comment by keianhzo** [comment]
@Softvision-RemusDranca If this is a regression, do you know what was the last version where it worked as expected?

**Comment by Softvision-RemusDranca** [comment]
@keianhzo I just tested builds starting from March and the issue is reproducible with those builds as well, so I think that this issue could be Pico related. 
I added the regression label because I was sure I haven't encountered it in the past, just starting from this week.
However, there was an minor update for Pico which I installed this week, with some extra boundary options and maybe other updates or the transition could have been to fast in the past and only now is clearly visible.
Thanks for pointing this out!
